plugins/woocommerce-admin/client/order-attribution/order-attribution-install-banner.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { Button, Card, CardBody } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Text } from '@woocommerce/experimental';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import OrderAttributionInstallBannerImage from './order-attribution-install-banner-image';
+import useOrderAttributionInstallBanner from './use-order-attribution-install-banner';
+import './style.scss';
+
+export const OrderAttributionInstallBanner = ( {
+	bannerImage = <OrderAttributionInstallBannerImage />,
+} ) => {
+	const {
+		isOrderAttributionInstallBannerDismissed,
+		dismissOrderAttributionInstallBanner,
+		shouldShowBanner,
+	} = useOrderAttributionInstallBanner();
+
+	if ( ! shouldShowBanner || isOrderAttributionInstallBannerDismissed ) {
+		return null;
+	}
+
+	recordEvent( 'order_attribution_install_banner_viewed' );
+
+	return (
+		<Card
+			size="medium"
+			className="woocommerce-order-attribution-install-analytics-overview-banner"
+		>
+			<CardBody className="woocommerce-order-attribution-install-analytics-overview-banner__body">
+				<div className="woocommerce-order-attribution-install-analytics-overview-banner__image_container">
+					{ bannerImage }
+				</div>
+				<div className="woocommerce-order-attribution-install-analytics-overview-banner__text_container">
+					<div className="woocommerce-order-attribution-install-analytics-overview-banner__text-chip">
+						<Text
+							className="woocommerce-order-attribution-install-analytics-overview-banner__text-description"
+							as="p"
+							size="12"
+							align="center"
+						>
+							{ __( 'New', 'woocommerce' ) }
+						</Text>
+					</div>
+					<Text
+						className="woocommerce-order-attribution-install-analytics-overview-banner__text-title"
+						as="p"
+						size="16"
+					>
+						{ __(
+							'Discover what drives your sales',
+							'woocommerce'
+						) }
+					</Text>
+					<Text
+						className="woocommerce-order-attribution-install-analytics-overview-banner__text-description"
+						as="p"
+						size="12"
+					>
+						{ createInterpolateElement(
+							__(
+								'Understand what truly drives revenue with our powerful order attribution extension. Use it to track<br/>your sales journey, identify your most effective marketing channels, and optimize your sales strategy.',
+								'woocommerce'
+							),
+							{ br: <br /> }
```

Comment:
See my comment [here](https://github.com/woocommerce/woocommerce/pull/51672#issuecomment-2373604960) regarding UI

Reply:
Fixed in

https://github.com/woocommerce/woocommerce/pull/51672/commits/749fb4dd2a5a4e34933b14b888380db0bc8cf3b3#diff-3815cff3f1a08b694ce40b4a0699e0cf64d9f7d0b0f3694313ae78d0bb12f56bL67-R70

https://github.com/woocommerce/woocommerce/pull/51672/commits/749fb4dd2a5a4e34933b14b888380db0bc8cf3b3#diff-71e16bc648210308ae9a1f2e30c8e8e311fe24425077449417a2f94ca8b2aa9bR60